clients/include/testing_matmul.hpp

```diff
@@ -364,6 +364,16 @@ void check(hipStream_t                          stream,
     }
 }
 
+void check_bias_type(const Arguments& arg)
+{
+    if((arg.a_type == HIP_R_8F_E4M3_FNUZ || arg.a_type == HIP_R_8F_E5M2_FNUZ)
+        && (arg.b_type == HIP_R_8F_E4M3_FNUZ || arg.b_type == HIP_R_8F_E5M2_FNUZ))
+    {
+        CHECK_SUCCESS((arg.bias_type == arg.scale_type || arg.bias_type == arg.d_type) && "For hipblastlt-bench/test, "
```

Comment:
For 8bit float in/ fp32 out case, the default bias type is FP16 which is not equal to scale_type and d_type.

Reply:
Removed this check for this PR as per our discussion in today's meeting, Will extend the bias type in another PR.